fe/src/main/java/org/apache/doris/alter/MaterializedViewHandler.java

```diff
@@ -858,4 +1160,57 @@ public void cancel(CancelStmt stmt) throws DdlException {
             jobDone(rollupJob);
         }
     }
+
+    // There may be dependency relation between user batch input rollup index
+    // e.g, user input rollup index: r1 from r2,r2 from r3 ;the input order must be r3,r2,r1
+    // sort rollup index for two reasons:
+    // 1. successfully create rollup index
```

Comment:
This comment confuse me.
`sort rollup index` -> `successfully create rollup index` ?

Reply:
Actually, rollup index can't be created in catalog before its parent rollup created,so does rollup job runs;so we must first sort rollup index,then  create index work and run rollup job;
I'll change the comment for better understand